src/components/Header/NetworkCard.tsx

```diff
@@ -0,0 +1,235 @@
+import { Trans } from '@lingui/macro'
```

Comment:
@biocom this entire comp is new and affecting the chainId visualisation

Reply:
OK, just visual changes right (that's what I've seen thus far)?